cmd/cloud/dbt.go

```diff
@@ -0,0 +1,154 @@
+package cloud
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	cloud "github.com/astronomer/astro-cli/cloud/deploy"
+	"github.com/astronomer/astro-cli/cloud/deployment"
+	"github.com/astronomer/astro-cli/config"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	mountPath      string
+	dbtProjectPath string
+
+	DeployBundle = cloud.DeployBundle
+)
+
+const (
+	dbtDeployExample = `
+Specify the ID of the Deployment on Astronomer you would like to deploy this dbt project to:
+
+  $ astro dbt deploy <deployment ID>
```

Comment:
This concerns me a little because we are using `astro dbt deploy` which implicitly states that dbt is a resource in this case that we would be deploying to, but then we end up passing `deployment id` as the argument to the command. 
Also, this goes against the existing convention of passing the deploy type as a flag to `astro deploy`, like `astro deploy --dags`

I think we could go with one of the following:
`astro dbt deploy --deployment <deployment ID>`
`astro deploy --dbt <deployment ID>` (Because we already assume deploy is an action on the deployment implicitly, a probable course correction for 2.x 🙂 )

Might prefer 1, as I can see ourselves adding new commands in the dbt space from the notion doc

Reply:
Agree that it is inconsistent with the existing deploy command. It's more of a product thing. We want a dedicated top-level dbt command that we can show off to analytic engineers who build dbt projects and don't want to be concerned with the Airflow things like images or DAGs that their data engineers have set up for them. And yeah, we can use this structure for `astro dbt delete` and maybe others too.

I can switch the deployment id from an argument to a flag though. It would probably want to be `--deployment-id` to clearly distinguish from `--deployment-name`.